iiop-openjdk/src/main/java/org/wildfly/iiop/openjdk/IIOPRootDefinition.java

```diff
@@ -149,6 +149,15 @@
             .addAccessConstraint(IIOP_SECURITY_DEF)
             .build();
 
+    public static final AttributeDefinition SECURITY_ENABLED_PROTOCOLS = new SimpleAttributeDefinitionBuilder(
+            Constants.SECURITY_SSL_ENABLED_PROTOCOLS, ModelType.STRING, true)
+            .setAttributeGroup(Constants.SECURITY)
+            .setDefaultValue(new ModelNode("TLSv1,TLSv1.1,TLSv1.2"))
+            .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
+            .addAccessConstraint(SensitiveTargetAccessConstraintDefinition.SOCKET_BINDING_REF)
+            .addAccessConstraint(IIOP_SECURITY_DEF)
+            .build();
+
```

Comment:
We have recently changed version to 3.0 - it will be the default in WF11 so imo another bump is not necessary. I will add the transformer though.